libraries/libfc/include/fc/io/cfile.hpp

```diff
@@ -19,7 +19,11 @@
 namespace fc {
 
 namespace detail {
-   using unique_file = std::unique_ptr<FILE, decltype( &fclose )>;
+   static void close_file(FILE* f) {
```

Comment:
I believe `inline` would be better here than `static`.

Reply:
ah yep